models/Event.model.js

```diff
@@ -9,12 +9,21 @@ const eventSchema = new Schema(
       maxLength: [50, `must not exceed 50 characters in length`],
       trim: true,
     },
+    address: {
+      city: String,
+      postcode: String,
+      street: String
+    },
     location: {
-      address: String,
-      coordinates: {
-        lat: Number,
-        long: Number,
+      type: {
+        type: String,
+        enum: ['Point'],
+        default: 'Point'
       },
+      coordinates: {
+        type: [Number],
+        required: true
+      }
     },
```

Comment:
I don't understand the modification on location: 
No more lat long ?
What's type and enum 'Point' ?

Reply:
This is how GeoJSON does it.
https://thecodebarbarian.com/80-20-guide-to-mongodb-geospatial-queries